### PR TITLE
update the sample with ResponseFieldMappers.MAPPERS as in https://git…

### PR DESCRIPTION
**Pull request by martinbonnin**
…hub.com/apollographql/apollo-android/issues/198#issuecomment-280673334

### CODE REVIEW

File: .idea/codeStyleSettings.xml [hunk 1]
```diff
@@ -289,6 +289,9 @@
         <JavaCodeStyleSettings>
           <option name="DO_NOT_WRAP_AFTER_SINGLE_ANNOTATION" value="true" />
         </JavaCodeStyleSettings>
+        <MarkdownNavigatorCodeStyleSettings>
+          <option name="RIGHT_MARGIN" value="72" />
```
Comment (digitalbuddha) [inline .idea/codeStyleSettings.xml]: Is this intentional?

Comment (martinbonnin) [inline .idea/codeStyleSettings.xml]: Not sure, I installed the markdown plugin for android studio, I guess it added this. I can remove it if needed.

**Comment by digitalbuddha** [comment]
Yeah let's take it out. Seems specific to your setup. I'd be down for git ignoring the idea folder @sav007 thoughts?

**Comment by digitalbuddha** [comment]
failing 
```
/home/travis/build/apollographql/apollo-android/apollo-sample/src/main/java/com/example/apollographql/sample/SampleApplication.java:6: error: cannot find symbol
import com.example.ResponseFieldMappers;
                  ^
  symbol:   class ResponseFieldMappers
  location: package com.example
/home/travis/build/apollographql/apollo-android/apollo-sample/src/main/java/com/example/apollographql/sample/SampleApplication.java:35: error: cannot find symbol
            .withResponseFieldMappers(ResponseFieldMappers.MAPPERS)
                                      ^
  symbol:   variable ResponseFieldMappers
  location: class SampleApplication
Note: /home/travis/build/apollographql/apollo-android/apollo-sample/build/generated/source/apollo/com/example/FeedQuery.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
2 errors
:apollo-sample:compileDebugJavaWithJavac FAILED
```

**Comment by martinbonnin** [comment]
Yup, I just saw it's not compiling. Looking into it.

**Comment by martinbonnin** [comment]
It compiles now. It looks like it was travis cache indeed. Removing it adds ~1min or so I would say.

**Comment by martinbonnin** [comment]
Re: README.md, I changed it to use ResponseFieldMappers when needed too.

**Pull request merged**